packages/sdk/wallets/wallet-hsm/package.json

```diff
@@ -21,16 +21,17 @@
   },
   "dependencies": {
     "@celo/base": "1.0.0-dev",
+    "@types/asn1js": "^0.0.2",
+    "@types/secp256k1": "^4.0.0",
+    "@types/debug": "^4.1.5",
     "eth-lib": "^0.2.8",
     "ethereumjs-util": "^5.2.0",
     "asn1js": "^2.0.26",
     "elliptic": "6.5.3",
     "secp256k1": "^4.0.0"
   },
   "devDependencies": {
-    "@celo/connect": "1.0.0-dev",
-    "@types/asn1js": "^0.0.2",
-    "@types/secp256k1": "^4.0.0"
+    "@celo/connect": "1.0.0-dev"
```

Comment:
I think that we are not even using this. We could remove this too.

Reply:
Done.